src/MetricsUploader/CaptureMetric.cpp

```diff
@@ -63,6 +67,19 @@ bool CaptureMetric::SendCaptureCancelled() {
 bool CaptureMetric::SendCaptureSucceeded(std::chrono::milliseconds duration_in_milliseconds) {
   capture_data_.set_duration_in_milliseconds(duration_in_milliseconds.count());
   status_code_ = OrbitLogEvent_StatusCode_SUCCESS;
+
+  if (file_path_.empty()) {
+    ERROR("Unable to determine capture file size for metrics. file path is empty");
```

Comment:
Nit: Maybe "File path is empty."  Capitalize the initial letter of "File".

Reply:
done